hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -402,7 +449,7 @@ def arrays(
                 .map((dtype.str + "[{}]").format)
                 .flatmap(lambda d: arrays(d, shape=shape, fill=fill, unique=unique))
             )
-        elements = from_dtype(dtype)
+        elements = from_dtype(dtype, **(elements or {}))
```

Comment:
The changes to `arrays` is not covered by the tests. It would be good to at least test the documented

```python
arrays(floating_dtypes(), shape, elements={"min_value": -10, "max_value": 10})
```

Reply:
Added two tests, one with inclusive and one with exclusive endpoints.